### ISSUE TITLE
React + Webpack Aliases + Testing = Help Fixing a Javascript bug

### ISSUE THREAD

**Issue by bmorelli25** [bmorelli25/ReactWeather#35]
Hello [Hacktoberfest](https://hacktoberfest.digitalocean.com/)!

Are you interested in React JS, Node JS, APIs, HTML, Javascript, Webpack, Express, and/or SASS/CSS? Then you've come to the right project!

This web app allows a user to type in a location, then pulls weather data from [OpenWeatherMap's API](http://openweathermap.org/) and returns it to the user. **[Live Demo](http://still-springs-11020.herokuapp.com/)**

**If this issue doesn't interest you, but there are other issues, bugs, enhancements you would like to make, please feel free to open up a new Issue and we can get a discussion started. I want everyone to be able to contribute!**

Your task: Webpack Aliases are broken in testing. Enhance your React and Webpack knowledge by fixing this bug! The details:

[What are Aliases?](https://webpack.github.io/docs/configuration.html#resolve-alias) You can find the aliases in this project in the `webpack.config.js` file. One example of using an alias would be in `Nav.jsx`. 

Instead of:
`var IPInfo = require('../api/ipInfo.jsx');`

You simply have to:
`var IPInfo = require('ipInfo');`

The problem (I believe) lies in the fact that `webpack.config.js` is not being required prior to tests being run. Thus, none of the aliases that are set up are accessed. In order to allow testing to work again, I have temporarily removed all Aliases from the project in PR #34. Please see the specific code changes for more information.

Don't hesitate to reach out with questions, or if you need help understanding the code!

_PS: This project utilizes a `.editorconfig` file. This helps developers define and maintain consistent coding styles between different editors and IDEs. Please ensure your editor is set up to use the `.editorconfig` file. For more information, check out [editorconfig.org](http://editorconfig.org/)._

**Comment by tetreault** [comment]
I know I still have [#14](https://github.com/bmorelli25/React-Weather-App/issues/14) on my plate still but once I finish the component stuff I'd be happy to lend a hand with this.

**Comment by sanketmaru** [comment]
Hey @bmorelli25 is any work pending on this ? I would be happy to pick it up ?

**Comment by tetreault** [comment]
Hey @sanketmaru feel free to take over on this. Sadly I've become inundated with projects at work so won't really have much free time for a while.

**Comment by sanketmaru** [comment]
@tetreault sure will start on this then.

**Comment by sanketmaru** [comment]
@bmorelli25 I am trying to use mocha-webpack(https://github.com/zinserjan/mocha-webpack) , with the configuration such as 
mocha-webpack --webpack-config webpack.config-test.js \"app/tests/*_/_.spec.js\""
Created a seperate config-test.js for this.

One of the test is failing when i run the npm test. Please see the screenshot. 

Thoughts on if we can use mocha-webpack ? I am figuring out if we can provide multiple path to test with the command.

![npm-test](https://cloud.githubusercontent.com/assets/7148430/19410624/74068b8e-930e-11e6-8658-9e1d21d3cef4.png)

@bmorelli25 @tetreault thoughts on this ?
